### PR TITLE
Gives clockcultists a usable UI and makes it not crash the server on win, so they can have a final goodbye round before being obliterated from the code completely.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Stops some runtimes, stops the server from crashing and gives clockies a (basic and ugly) UI to use their scriptures.

This is no where near a fix, and the gamemode is too broken to fix at this point, but it lets admins run the gamemode without the server crashing (probably), so we can run a few final rounds of the clockies.

By any means this does not iron out the major flaws with the actual design of the gamemode, it just makes it playable (I don't see this being in for much longer after this).

## Why It's Good For The Game

Just lets clock cult run a few times before its removal (if admins choose to do so).

## Changelog
:cl:
fix: Clock cultists have a UI and ratvar doesn't turn the server to cogs.
code: Makes tgui/bin/tgui-build.bat time out after 90 seconds because it's hard to read errors in 9 seconds.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
I know it's a broken gamemode, this makes it playable not maintanable

**Comment by PowerfulBacon** [comment]
I'll delete clock cult gamemode in another PR after this

**Comment by Crossedfall** [comment]
This is a pretty pointless pr. I'm going with no on this one.

**Comment by PowerfulBacon** [comment]
> 
> 
> This is a pretty pointless pr. I'm going with no on this one.

alright check out the removal one then

File: tgui/bin/tgui-build.bat [hunk 1]
```diff
@@ -2,4 +2,4 @@
 cd "%~dp0\.."
 call yarn install
 call yarn run build
-timeout /t 9
+timeout /t 90
```
Comment (Crossedfall) [inline tgui/bin/tgui-build.bat:5]: If you build through vscode or another terminal session this isn't an issue. It'd be better to have this remain as a lower number

Comment (PowerfulBacon) [reply tgui/bin/tgui-build.bat:5]: You can build TGUI through VSCODE? how?

**Pull request closed without merging**